packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method-form.jsx

```diff
@@ -68,12 +68,12 @@ const AddPaymentMethodForm = ({ formik_ref, should_show_separated_footer = false
                                     )}
                                 </Field>
                                 {my_profile_store.selected_payment_method_fields &&
-                                    my_profile_store.selected_payment_method_fields.map((payment_method_field, key) => {
+                                    my_profile_store.selected_payment_method_fields.map(payment_method_field => {
                                         return (
                                             <Field
                                                 name={payment_method_field[0]}
                                                 id={payment_method_field[0]}
-                                                key={key}
+                                                key={payment_method_field[0]}
```

Comment:
Is `payment_method_field[0]` a dynamic value?

Reply:
Hello @matin-binary,

Yes it is dynamic `payment_method_field` is an object like in the screenshot attached
![p2p](https://user-images.githubusercontent.com/104334373/183856557-9354b2fb-f95c-4a2a-b086-984470dc9edf.png)